packages/app/src/server/bot.ts

```diff
@@ -8,65 +9,72 @@ export type Option = {
   id: string;
 };
 
-let bot: null | octokit = null;
+let bot: null | Octokit = null;
 
 export function setup() {
-  bot = new octokit({
-    baseUrl: process.env.GITHUB_API_URL
-  });
-
+  const option: ConstructorParameters<typeof Octokit>[0] = {
+    baseUrl: process.env.GITHUB_API_URL,
+  };
   if (process.env.GITHUB_USER_NAME && process.env.GITHUB_PASSWORD) {
-    // TODO: make use of 2FA
-    bot.authenticate({
-      type: 'basic',
+    option.authStrategy = createBasicAuth;
+    option.auth = {
```

Comment:
I had to force push because it wasn't working well. Here is the diff.

```diff
diff --git a/packages/app/src/server/bot.ts b/packages/app/src/server/bot.ts
index dc1171d..c64c8f5 100644
--- a/packages/app/src/server/bot.ts
+++ b/packages/app/src/server/bot.ts
@@ -16,12 +16,17 @@ export function setup() {
     baseUrl: process.env.GITHUB_API_URL,
   };
   if (process.env.GITHUB_USER_NAME && process.env.GITHUB_PASSWORD) {
-    // TODO: make use of 2FA
     option.authStrategy = createBasicAuth;
     option.auth = {
-      type: 'basic',
       username: process.env.GITHUB_USER_NAME,
       password: process.env.GITHUB_PASSWORD,
+      token: {
+        scopes: ['public_repo'],
+      },
+      on2Fa: () => {
+        // TODO: make use of 2FA
+        throw new Error('2FA not supported');
+      },
     };
   }
 ```